src/Compilers/CSharp/Portable/Symbols/AnonymousTypes/PublicSymbols/AnonymousType.DelegatePublicSymbol.cs

```diff
@@ -16,9 +16,20 @@ internal sealed class AnonymousDelegatePublicSymbol : AnonymousTypeOrDelegatePub
         {
             private ImmutableArray<Symbol> _lazyMembers;
 
-            internal AnonymousDelegatePublicSymbol(AnonymousTypeManager manager, AnonymousTypeDescriptor typeDescr) :
+            /// <summary>
+            /// This member does not participate in equality because it is not reflecting any semantic aspect of the symbol.
+            /// It is only used to determine if we need to check for <see cref="MessageID.IDS_ParamsCollections"/>
+            /// feature availability, which happens if this field is set to 'true'. 
+            /// If in the process of merging equivalent types, the one with 'false' wins over the one with 'true'.
+            /// That is fine, because that means that the feature availability check is performed on a
```

Comment:
Nit:
```suggestion
            /// If in the process of merging equivalent types, the one with 'false' wins over the one with 'true',
            /// that is fine, because that means that the feature availability check is performed on a
```